src_c/polygon.c

```diff
@@ -140,23 +187,31 @@ pgPolygon_FromObject(PyObject *obj, pgPolygonBase *out)
 
         if (length >= 3) {
             Py_ssize_t i;
+            Py_ssize_t i2;
```

Comment:
i am not sure about that but the `s` in ssize is for signed, you need to use unsigned int64 because memory addresses are unsigned

Reply:
hmmm... what you are saying is that since that `i2` variable is not used here in particular for python function implementation, I should use size_t, right?